src/packages/admin-ui-components/src/detail-panel/component.tsx

```diff
@@ -181,6 +62,17 @@ const DetailField = ({ field }: { field: EntityField }) => {
 	);
 };
 
+const CustomField = ({ field }: { field: CustomField }) => {
```

Comment:
Should this also be in the fields folder?

Reply:
On this one, it's actually an alternative to `DetailField`, which is also in this file, e.g. the things in the folder are the actual fields, where these can more be called "wrappers"... What do you reckon?

Reply:
sounds good 👍